jtelegrambotapi-core/src/main/java/com/jtelegram/api/user/User.java

```diff
@@ -24,7 +29,7 @@
      * @return The full name
      */
     public String getFullname() {
-        return new StringBuilder(firstName).append(" ").append(lastName).toString();
+        return Stream.of(firstName, lastName).filter(Objects::nonNull).collect(Collectors.joining(" "));
```

Comment:
Is this really the best or even most simple way to do this? Couldn't a simple if statement be sufficient instead of a O(2n) function?

Reply:
Do you have any proof for that? I'm not the most familiar with how streams internally work but I'd be surprised to see them implement basic operations like this just as efficiently as an if statement. I just feel like in this case, using streams is a bit over the top but if you insist I'm fine to keep it as is.

Reply:
Technically since it is two inputs its actually just that. But streams are way too overkill for this lol. Feel free to use it though! Showcases interesting use of stream and filter architecture.